packages/react-google-maps-api/src/components/drawing/MarkerWithLabel.tsx

```diff
@@ -33,7 +34,13 @@ const markerWithLabelUpdaterMap = {
   ...updaterMap,
 }
 
-const MarkerWithLabel: FC<MarkerWithLabelProps> = props => {
+const MarkerWithLabel: FC<MarkerWithLabelProps> = ({
+  labelAnchor,
```

Comment:
Something also seems to be wrong with the eslint setup, I get warnings where I should not get them.

Reply:
can you attach a screenshot of warnings?

I will continue to work on alpha version this week, tried alpha in codesandbox example and it fails.